File: android/SmartIRHub/app/src/main/java/com/ms8/smartirhub/android/_tests/dev_playground/remote_layout/asymmetricgridview/LinearLayoutPoolObjectFactory.java
package com.ms8.smartirhub.android._tests.dev_playground.remote_layout.asymmetricgridview;

import android.content.Context;
import android.widget.LinearLayout;

public class LinearLayoutPoolObjectFactory implements PoolObjectFactory<LinearLayout> {

  private final Context context;

  public LinearLayoutPoolObjectFactory(final Context context) {
    this.context = context;
  }

  @Override
  public LinearLayout createObject() {
    return new LinearLayout(context, null);
  }
}

File: android/SmartIRHub/app/src/main/java/com/ms8/smartirhub/android/_tests/dev_playground/remote_layout/asymmetricgridview_k/LinearLayoutPoolObjectFactory.kt
package com.ms8.smartirhub.android._tests.dev_playground.remote_layout.asymmetricgridview_k

import android.content.Context
import android.widget.LinearLayout

class LinearLayoutPoolObjectFactory(private val context: Context) : PoolObjectFactory<LinearLayout> {

    override fun createObject() = LinearLayout(context, null)
}